aces/Loris
# [Create Timepoint] updated module help text

updated the help text

#### Testing instructions

1. edit copy if necessary
2. check screenshot for rendering in LORIS

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576#issuecomment-556905146

![Screenshot 2019-11-24 18 57 27](https://user-images.githubusercontent.com/37813606/69503882-46232300-0eec-11ea-8a4a-ef0ecd02e052.png)
